docs/src/upgrade/release-3.3.x.asciidoc

```diff
@@ -29,6 +29,15 @@ Please see the link:https://github.com/apache/tinkerpop/blob/3.3.1/CHANGELOG.asc
 
 === Upgrading for Users
 
+==== Gremlin-Python Core Types
```

Comment:
You can remove this - no need to duplicate it. We direct users to 3.2.x upgrade notes as being inclusive of the 3.3.x ones.